src/Sylius/Bundle/ApiBundle/Validator/Constraints/EligibleProductVariantToCart.php

```diff
@@ -16,7 +16,7 @@
 use Symfony\Component\Validator\Constraint;
 
 /** @experimental */
-final class AddingEligibleProductVariantToCart extends Constraint
+final class EligibleProductVariantToCart extends Constraint
```

Comment:
```suggestion
final class EligibleProductVariantInCart extends Constraint
```

You should mention this change in UPGRADE file as this constraint has been added in `1.9`